src/applications/disability-benefits/2346/config/form.js

```diff
@@ -81,6 +94,87 @@ const formConfig = {
         },
       },
     },
+    OrderSuppliesChapter: {
+      title: formChapters.orderSupplies,
+      pages: {
+        [formPages.orderSuppliesPage]: {
```

Comment:
Is there a clear benefit to setting up a page like this? Currently this first page gets assigned the string `Personal Details` as its property name, which isn't technically a valid identifier.

Reply:
This is driven by design.